prettypandas/styler.py

```diff
@@ -46,8 +47,15 @@ def apply_pretty_globals():
 
 
 Formatter = namedtuple("Formatter", "subset, function")
-
-
+remove_nan_formatter = lambda x: '' if x == np.nan else x
+
+def PrettyPandasNoIndex(dataframe, table_styles = [], *args, **kwargs):
+    return PrettyPandas(dataframe, 
+                        *args, 
+                        table_styles = table_styles + [{'selector': '.row_heading', 'props': [('display', 'none')]}, 
+                                                       {'selector': '.blank', 'props': [('display', 'none')]}],
+                        **kwargs)
+                        
```

Comment:
I prefer using a class to keep common conventions of functions being snake_case and classes being CammelCase. Plus if the upstream pandas style changes there's a cleaner separation of code.